### DISCUSSION TITLE
2023-10-25 Data Architecture Meeting

### DISCUSSION THREAD

**Discussion by glopesdev** [SainsburyWellcomeCentre/aeon_mecha#267] [Meeting Agendas]
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/347
- https://github.com/SainsburyWellcomeCentre/aeon_mecha/pull/264
  - Which attributes are displayed in web interface versus their web API?
    - Need to clarify which web interface form entries map to each of the pyrat web api fields
    - `#colony` table is replaced with join between `subject` and `#subject_detail`
      - additional tables for each subject:
        - `subject_weight`
        - `subject_comment`
        - `subject_procedure`
     - reference weight needs to be replaced with a manual attribute, one suggestion is to take the latest weight from pyrat as the manual reference weight by querying the `subject_weight` table
     - consider filtering out subjects by experiment type and status
     - consider triggering DJ refresh from pyrat on workflow start (need to figure out exact steps for this, e.g. insert a value in a table)
     - all pyrat data is accessible through the `responsible_id` attribute, currently only @jkbhagatio is registered, other users will need to register themselves to get their data pulled. only entries tagged with `"label": "Project"` and `"content": "Aeon"` attributes will be pulled (as described in https://github.com/SainsburyWellcomeCentre/aeon_mecha/issues/263)
- https://github.com/SainsburyWellcomeCentre/aeon_mecha/issues/266
- End-to-end testing of Bonsai-Sleap pose tracking
- Refactor `aeon.qc.video` to work on all cameras of any schema

**Comment by jkbhagatio** [comment]
we should have test aeon subjects as well for running exps (to replace Dario and Lory =p)
